web/static/js/app.jsx

```diff
@@ -217,7 +213,7 @@ var SearchBox = React.createClass({
     }
   },
 
-  onChange: function(e) {
```

Comment:
Right. The function _does_ receive this argument so it seems helpful to state that. I don't feel super strongly but would probably disable this lint rule if it were my project.